packages/contracts/contracts/engine/V2/GenArt721Minter_PBAB.sol

```diff
@@ -251,6 +273,14 @@ contract GenArt721Minter_PBAB is ReentrancyGuard {
                 )
             ) != keccak256(abi.encodePacked("ETH"))
         ) {
+            uint256 pricePerTokenInWei = genArtCoreContract
```

Comment:
For gas efficiency, we can replace that `abi.encodePacked("ETH")` stuff above with just `if (configuredCurrencyAddress  != address(0)) {...}`, since you already loaded sufficient information in the new checks ✅

Reply:
Awesome, updated!